Core/src/IfcGeometryConverter/SolidModelConverter.h

```diff
@@ -174,9 +182,69 @@ namespace OpenInfraPlatform
 
 					BLUE_LOG(warning) << "Geometry conversion for IfcSectionedSolidHorizontal not implemented.";
 					// TO DO: implement, check for formal propositions. 
+                 
 
-				 //endif sectioned_solid_horizontal
-			}
+				//check dimensions and correct attributes sizes
+                if(vec_cross_sections.size() != vec_cross_section_positions.size())
+				{
+					std::cout << "CrossSections and CrossSectionsPositions are not equal in size " << std::endl;
+					return;

```

Comment:
this should be an exception
```suggestion
					throw oip::InconsistentModellingException("CrossSections and CrossSectionsPositions are not equal in size.");
```

Reply:
resolved in commit ccd8948